keras/backend/tensorflow_backend.py

```diff
@@ -1307,6 +1339,22 @@ def clip(x, min_value, max_value):
     return tf.clip_by_value(x, min_value, max_value)
 
 
+def clipnorm(x, max_value):
```

Comment:
Ah sorry that was an artifact from another PR. I have removed it now. You will have to squash and merge.